### PR TITLE
Replace direct references to 'kubectl' and 'docker' executables with …

### PR DESCRIPTION
**Pull request by tbarnes-us**
**These changes include:**

- Updating sample & test java and scripts.
- Changing references to docker or kubectl to KUBERNETES_CLI and WLSIMG_BUILDER respectively.
- Changing method names and comment text that include variations of 'kubectl' or 'docker' to something more generic.
- Adding a `./validateCLI.sh` build check in 'pom.xml' to check for invalid direct use of 'docker', 'Docker', 'DOCKER', or 'kubectl', 'Kubectl', or 'KUBECTL'. This script can also be run stand-alone.

**These changes deliberately do _not_ include:**

- Updating documentation.

- Updating lines that are part of a comment (many were changed, but not all).

- Updating the more specialized KUBECTL usages, such as:
-- the 'KUBECTL_VERSION' global
-- the 'kubectl.kubernetes.io.last-applied-configuration' annotation

- Updating the more specialized docker usages, such as:
--  var name 'dockerFile' and filename 'Dockerfile'
--  script name create-docker-credentials-secret.sh, buildDockerImage.sh, ...
--  default secret name 'docker-store'

See the "sed filters" in `./validateCLI.docker.dat` and `./validateCLI.kubectl.dat` for the full list.

**See also:**

WebLogic Image Tool pull: https://github.com/oracle/weblogic-image-tool/pull/393
Related Issue: https://github.com/oracle/weblogic-kubernetes-operator/issues/1644

### CODE REVIEW

**Review by rjeberhard — APPROVED** [review]

**Review by anpanigr — COMMENTED** [review]
Can you please put the Jenkin Sequential/Parallel results on PR description.

File: kubernetes/samples/scripts/create-weblogic-domain/model-in-image/ingresses/traefik-ingress-sample-domain1-admin-server.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2020, 2021, Oracle and/or its affiliates.
+# Copyright (c) 2020, 2022, Oracle and/or its affiliates.
```
Comment (lennyphan) [inline kubernetes/samples/scripts/create-weblogic-domain/model-in-image/ingresses/traefik-ingress-sample-domain1-admin-server.yaml:1]: Does copyright need to be changed if there isn't any change to this file?

File: kubernetes/samples/scripts/create-weblogic-domain/model-in-image/ingresses/traefik-ingress-sample-domain2-cluster-cluster-1.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2020, 2021, Oracle and/or its affiliates.
+# Copyright (c) 2020, 2022, Oracle and/or its affiliates.
```
Comment (lennyphan) [inline kubernetes/samples/scripts/create-weblogic-domain/model-in-image/ingresses/traefik-ingress-sample-domain2-cluster-cluster-1.yaml:1]: copyright change although no change to contents?

**Review by lennyphan — APPROVED** [review]

**Comment by rjeberhard** [comment]
@tbarnes-us, what's the status of this change? I can help with the merge, but I'm not sure whether it's otherwise ready to go.

**Pull request merged**